xapian-core/tests/harness/backendmanager_multi.h

```diff
@@ -39,7 +40,8 @@ class BackendManagerMulti : public BackendManager {
     /// The path of the last writable database used.
     std::string last_wdb_path;
 
-    BackendManager* sub_manager;
+    // vector of sub_managers.
+    std::vector<BackendManager*> sub_managers;
```

Comment:
You should `#include <vector>` if you're using `std::vector` (I guess some header being included is including `<vector>` for you, but you shouldn't rely on such things.